server/handlers.go

```diff
@@ -160,14 +174,44 @@ func (svr *Server) handleGetEigenDADispersalBackend(w http.ResponseWriter, _ *ht
 // =================================================================================================
 
 // handlePostStdCommitment handles the POST request for std commitments.
+// parseEncodingQueryParamType parses the encoding query parameter
+func parseEncodingQueryParamType(w http.ResponseWriter, r *http.Request) (commitments.EncodingType, bool, error) {
```

Comment:
what is the bool returned for?

Reply:
indicating whether the encoding query param was provided or not. Retrospectively we can enforce the misusage invariant (i.e, `v1Backend=true && encodingQueryParam=true`) in this function directly vs propagating to multiple callers. will just require making the function self referential or accessor to the server

Reply:
04dba8c7e4368cfd65fdfc4b9bb95f480a8ec008